pkg/virtctl/imageupload/imageupload.go

```diff
@@ -58,7 +58,8 @@ const (
 	// PodReadyAnnotation tells whether the uploadserver pod is ready
 	PodReadyAnnotation = "cdi.kubevirt.io/storage.pod.ready"
 
-	uploadRequestAnnotation = "cdi.kubevirt.io/storage.upload.target"
+	uploadRequestAnnotation         = "cdi.kubevirt.io/storage.upload.target"
+	forceImmediateBindingAnnotation = "cdi.kubevirt.io/storage.bind.immediate.requested"
```

Comment:
Should we just reference these from the [CDI repo](https://github.com/kubevirt/containerized-data-importer/blob/5fa334b1c50cabd6ec523a0dd167ec1cd8a8ce02/pkg/controller/util.go#L109)?

Reply:
Yes and no.

This is a great idea to just reference something from API, but this annotation is an internal CDI API. On one hand, I would not like to include all internal utils of CDI. On the other, CDI is already tightly integrated with Kubevirt in a few places. I need to think deeper about this. Maybe `storage.bind.immediate.requested` should be promoted to a field in a DataVolume.

BTW: There is a plan to move CDI APIs to distinct repo. That way projects like kubevirt or other could import just the CDI API from API repo as dependency. This will make it more apparent which things are considered public API. 

If I understand correctly other annotations here, are the legacy api, from the time when there was not a  DataVolume API.

Reply:
> but this annotation is an internal CDI API

So, if I get this right, the annotation itself is a public API, but its definition resides within an internal CDI package.
To that end, it sounds like a good plan to have it eventually move to a CDI API repo.